### PR TITLE
add content for sp22 and switch to it

### PR DESCRIPTION
**Pull request by ngwattcos**
Add content for SP22 semester and switch to it. Adds contacts to Enoch, Scott, and Jason, Mentions no classes during Spring Break.

### CODE REVIEW

File: versioned_docs/version-2022sp/setup-environment.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+id: setup-environment
+title: Setup your development environment
+---
+
+## Install Node.js
+
+Go to [this website](https://nodejs.org/en/download/) and follow the instructions.
+
+For consistency, we will use Node LTS (currently, as of Spring 2021, this is Node 14).
```
Comment (Enochen) [inline versioned_docs/version-2022sp/setup-environment.md:10]: Let's update this to 16

File: versioned_sidebars/version-2022sp-sidebars.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      },
+      {
+        "type": "doc",
+        "id": "version-2022sp/setup-editor"
+      }
+    ],
+    "Lectures": [
+      {
+        "type": "doc",
+        "id": "version-2022sp/lecture1"
```
Comment (Enochen) [inline versioned_sidebars/version-2022sp-sidebars.json:20]: Let's leave out broken links like these

**Review by Enochen — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2022sp/setup-environment.md (hunk 1 above)
Comment (ngwattcos) [reply versioned_docs/version-2022sp/setup-environment.md]: updated

File: versioned_sidebars/version-2022sp-sidebars.json (hunk 2 above)
Comment (ngwattcos) [reply versioned_sidebars/version-2022sp-sidebars.json]: removed broken links to lectures, assignments

**Review by Enochen — APPROVED** [review]
Looks great, thanks for updating!

**Pull request merged**